cranelift/codegen/src/isa/arm64/abi.rs

```diff
@@ -559,6 +563,7 @@ pub struct ARM64ABICall {
     uses: Set<Reg>,
     defs: Set<Writable<Reg>>,
     dest: CallDest,
+    loc: ir::SourceLoc,
```

Comment:
For line debuginfo all instructions would need a SourceLoc.

Reply:
Yep, we haven't really built out proper debuginfo support yet (of course there's a lot more too, like mapping values to registers). I'm assuming Ben added this now just to get reasonable stack traces...?

Reply:
Resolving: it's not solving the general case indeed, but just does what's needed for the new Spidermonkey integration (see PR top-level comments).